j/jackson-annotations/jackson-annotations_ubi_9.3.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package : jackson-annotations
+# Version : 2.17.0
+# Source repo : https://github.com/FasterXML/jackson-annotations/tree/2.17
```

Comment:
@pratibh-ibm Source repo should be github base url and not branch url. Try keep indentation in this form like 
![image](https://github.com/ppc64le/build-scripts/assets/82815888/374b9520-e307-4b77-9d01-751c80e47eb5)